lua/spd/server/sv_spd.lua

```diff
@@ -10,17 +10,10 @@ hook.Add("EntityRemoved", "spdEntityRemovedHook", spdEntityRemoved)
 local function spdEntityTakeDamage(ent, dmg)
 	local entOwner = ent:CPPIGetOwner()
 
-	if not IsValid( entOwner ) then
-		return
-	end
-
-	if GetConVar("spd_enabled"):GetInt() == 0 then
-		return
-	end
-
-	if ent:GetClass() ~= "prop_physics" then
-		return
-	end
+	if not IsValid( entOwner ) then return end
+	if not IsValid( ent ) then return end
```

Comment:
This is new functionality, breaks out if `ent` is invalid. I don't see why this would be a problem, but it could be a breaking change